restcomm/restcomm.commons/src/main/java/org/restcomm/connect/commons/dao/Sid.java

```diff
@@ -31,12 +31,59 @@
  */
 @Immutable
 public final class Sid {
+
+    private static String CALL_SID_STRING = "ID[a-zA-Z0-9]{32}-CA[a-zA-Z0-9]{32}";
     public static final Pattern pattern = Pattern.compile("[a-zA-Z0-9]{34}");
-    public static final Pattern callSidPattern = Pattern.compile("ID[a-zA-Z0-9]{32}-CA[a-zA-Z0-9]{32}");
+    public static final Pattern callSidPattern = Pattern.compile(CALL_SID_STRING);
+
     private final String id;
 
     public enum Type {
-        ACCOUNT, APPLICATION, ANNOUNCEMENT, CALL, CLIENT, CONFERENCE, GATEWAY, INVALID, NOTIFICATION, PHONE_NUMBER, RECORDING, REGISTRATION, SHORT_CODE, SMS_MESSAGE, TRANSCRIPTION, INSTANCE, EXTENSION_CONFIGURATION, GEOLOCATION, ORGANIZATION, PROFILE
+        CALL(null, CALL_SID_STRING),
```

Comment:
not sure how this Sid refactoring is related to original PR issue, but im fine with it

Reply:
investigating some issues with this refactoring